features/step_definitions/kata.rb

```diff
@@ -166,7 +166,12 @@
     logger.info("There's already an existing 'kataconfig' resuing it...")
     project(kata_ns)
     step %Q/I switch to cluster admin pseudo user/
+    # make sure kata-operator is running first before installing the kataconfig
+    step %Q/a pod is present with labels:/, table(%{
+      | control-plane=controller-manager |
+    })
```

Comment:
This check seems really generic.  Are you sure that this check is specific to the sandboxed-container-operator and would not be satisfied falsely by something else?

Reply:
@cpmeadors yes, that's the first and only pod with that label in the project.  We could request the pod to have a more specific label, but that has to be done kata itself.